### ISSUE TITLE
Cloud logging

### ISSUE THREAD

**Issue by ivomarino** [ttssdev/appflow#53]
This is definitely a hot topic, would be great to add some kind of https://www.loggly.com support or similar, this way logs could be shipped directly to a target log server or cloud solution. It's important for devs in order to be able to get their logs in case of troubles or performance analysis.

**Comment by grappler** [comment]
Also possible with https://papertrailapp.com/

**Comment by ivomarino** [comment]
Another candidate: https://www.datadoghq.com/product/

**Comment by ivomarino** [comment]
@maschli we should evaluate prices for those services and compare.

**Comment by maschli** [comment]
@ivomarino yep, but before we have to define our criteria (must, nice-to-have etc.), befor we are evaluate a tool.
